less/alerts.less

```diff
@@ -0,0 +1,33 @@
+// Alerts
+// --------------------------------------------------
+//
+
+.alert {
+  padding: @alert-padding;
+  margin-bottom: @line-height-computed;
+  border: 1px solid transparent;
+  border-radius: @alert-border-radius;
+
+ // Improve alignment and spacing of inner content
+  > p,
+  > ul {
+    margin-bottom: 10px;
+  }
+
+  > p + p {
+    margin-top: 5px;
+  }
+}
+
+.alert-dismissable, // The misspelled .alert-dismissable was deprecated in 3.2.0.
+.alert-dismissible {
+  padding-right: (@alert-padding + 20);
```

Comment:
Hm, this is correct, but it's unfortunate that we have to add duplicate statements all over (due to how CSS works).

I think we made a mistake by creating separate files with our changes. It'd be much cleaner to just modify the original bootstrap source. It's still easy to see the modifications and generate a patch with git. But let's not worry about that here.